docs/source/faq.rst

```diff
@@ -99,5 +99,14 @@ Frequently Asked Questions
         case you could also report your problem on our `issue tracker`_ and
         work with the dask development team to improve our scheduling policies.
 
+5. **How does Dask serialize functions?**
+
+When operating with the single threaded or multithreaded scheduler no function
+serialization is necessary.  When operating with the distributed memory or
+multiprocessing scheduler Dask uses cloudpickle_ to serialize functions to send
+to worker processes.  cloudpickle supports almost any kind of function,
+including lambdas, closures, partials and functions defined interactively.
```

Comment:
It is worth mentioning here how one might debug/troubleshoot a function that isn't serializable via cloudpickle or test whether it is? Or somewhere else (e.g., diagnostics/troubleshooting page)?